devops/iam-did-auth-proxy-dev/values.yaml

```diff
@@ -273,9 +273,18 @@ did-auth-proxy-helm:
         nginx.ingress.kubernetes.io/proxy-body-size: 50m
         external-dns.alpha.kubernetes.io/cloudflare-proxied: "true"
       hostname: gp4btc-backend-proxy-dev.energyweb.org
+      annotations: 
+        kubernetes.io/ingress.class: alb
+        alb.ingress.kubernetes.io/target-type: ip
+        alb.ingress.kubernetes.io/scheme: internet-facing
+        alb.ingress.kubernetes.io/listen-ports: '[{"HTTPS":443}]'
+        external-dns.alpha.kubernetes.io/cloudflare-proxied: 'true'
+        alb.ingress.kubernetes.io/certificate-arn: 'arn:aws:acm:us-west-2:984870885661:certificate/3d6211f2-ec40-4a44-9948-49b77b22f50c'
+        alb.ingress.kubernetes.io/actions.ssl-redirect: '{"Type": "redirect", "RedirectConfig": { "Protocol": "HTTPS", "Port": "443", "StatusCode": "HTTP_301"}}'
+      hostname: gp4btc-backend-proxy.energyweb.org
```

Comment:
@JGiter This seems wrong. This is duplicating the `annotations` section and the `hostname` is wrong